### DISCUSSION TITLE
Suspended courses

### DISCUSSION THREAD

**Discussion by KevinUrquijo08** [chamilo/chamilo-lms#5780] [General]
Good morning, my chamilo accounts and courses were recently suspended due to inappropriate use of it, due to the number of users and courses I had created. I would like to know if there is any way to recover the information about the courses that have been restricted for me.
I appreciate your contributions.

**Comment by Coursenligne** [comment]
Could you be more precise about the environment?
A "normal" installation doesn't restrict at all.

**Reply by KevinUrquijo08** [reply]
We were using the virtual campus, and we created more than 2 courses, in addition we had many students, because we did not eliminate the previous ones, so we were suspended from the virtual campus and now the information of our courses is not available.
So we want to know if is possible to get the information back.

**Reply by Coursenligne** [reply]
I'm not sure about your virtual campus.
I know it's obvious for you, but not for me.
Let's go another way...
You have several ways to get a Chamilo Platform.
The first one is to install it on premises, your therefore create a server and install it.
For this (the most common) there is no restriction. I eliminate it!
You can be hosted. In such a condition it is like previously, but you have a service provider that gives you an amount of his server. This could be your installation (virtual campus truely means nothing for me).
If so, you may contact your service provider as He created the restriction so He can assist you in removing the irrelevant users and get back to a "normal use".
The third way I see is to contact Beeznest or a Chamilo official partner and they will get rid of your problem.

Anyway this should be a matter of configuration.
If you installed yourself the platform, I may (and I already did) help you even further more. In other cases, you should contact your provider. 
Depending on what they say I will better understand the meaning of virtual campus and may think of a better potential solution.

I hope this helps.
